src/main/java/org/csanchez/jenkins/plugins/kubernetes/volumes/workspace/WorkspaceVolume.java

```diff
@@ -36,6 +36,10 @@
 
     private static final long serialVersionUID = 5367004248055474414L;
 
+    // Builds a Volume model with the given name.require podName to generate pvc name
+    public abstract Volume buildVolume(String volumeName, String podName);
+
     // Builds a Volume model with the given name.
+    @Deprecated
     public abstract Volume buildVolume(String volumeName);
```

Comment:
Provide a default implementation

```
@Deprecated
public Volume buildVolume(String volumeName){
    throw new UnsupportedOperationException("could not build volume without podName");
}
```

so that you don't have to implement both signatures in `DynamicPVCWorkspaceVolume`